src/Microsoft.Diagnostics.DebugServices/ISymbolService.cs

```diff
@@ -18,9 +18,13 @@ public interface ISymbolService
         /// </summary>
         bool IsSymbolStoreEnabled { get; }
 
+        /// <summary>
+        /// The default symbol server URL (normally msdl) when not overridden in AddSymbolServer.
+        /// </summary>
+        string DefaultSymbolPath { get; set; }
```

Comment:
wouldn't `get` be enough?

Reply:
I wanted to be settable like the other properties (DefaultTimeout, DefaultRetryCount and DefaultSymbolCache).

Reply:
Feels interesting to set such data through the interface and not the instantiation. But that's ok

Reply:
You are right. It is a little weird to have the setters in interface for all the Default* props.  I'll think about it and probably push some changes.